cron/internal/worker/main.go

```diff
@@ -128,11 +129,21 @@ func processRequest(ctx context.Context,
 		if err := format.AsJSON2(&result, true /*showDetails*/, log.InfoLevel, checkDocs, &buffer2); err != nil {
 			return fmt.Errorf("error during result.AsJSON2: %w", err)
 		}
+		exportPath := fmt.Sprintf("%s/result.json", repo.URI())
+		exportCommitSHAPath := fmt.Sprintf("%s/%s/result.json", repo.URI(), result.Repo.CommitSHA)
```

Comment:
Could we also store the raw data results to `projectPath/raw.json` and `projectPath/commitSHA/raw.json`?

Reply:
I have fixed it. Thanks